pkgs/servers/foundationdb/default.nix

```diff
@@ -17,15 +17,18 @@
   toml11,
   jemalloc,
   doctest,
+  zlib,
+
+  nixosTests,
 }@args:
 
 let
   cmakeBuild = import ./cmake.nix args;
 in
 {
   foundationdb73 = cmakeBuild {
-    version = "7.3.42";
-    hash = "sha256-jQcm+HLai5da2pZZ7iLdN6fpQZxf5+/kkfv9OSXQ57c=";
+    version = "7.3.62";
+    hash = "sha256-tPB/jJbdC11wQvPOI5KtlpQgw9yVLBhxABGgDSBpwBU=";
```

Comment:
```suggestion
    hash = "sha256-fUyxV6oZdJOh0mv+uWz4hiNqyQHDR6hekTLS62XlPM8=";
```

Reply:
The odd-numbered releases have AVX enabled, we want the odd-numbered releases.